roles/nova-common/tasks/logging.yml

```diff
@@ -7,3 +7,6 @@
       - missingok
       - rotate 7
       - compress
+      - sharedscripts
+    scripts:
+      - postrotate: "/usr/sbin/service nova-novncproxy reload"
```

Comment:
This format looks different than what we used in another recent PR for scripts.  https://github.com/blueboxgroup/ursula/pull/1330/files#diff-d37d14194def27271d73456959f0f928R13